xml/upgrade-3.10-0.xsl

```diff
@@ -171,6 +175,45 @@
 </xsl:template>
 
 
+<!-- Resources -->
+
+<!--
+ Convert master resources to clones with promotable meta-attribute set to "true"
+ -->
+<xsl:template match="master">
```

Comment:
This needs to be moved to a later transformation, after the sorting has already happened, or needs to call for sorting explicitly. Otherwise, we won't match against the sorting template, and the option blocks within this resource won't get sorted.